baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -101,3 +102,17 @@ For example, if you have non-production requests that flow through the same serv
 ```
 baggage: isProduction=false
 ```
+# Mutating baggage
+A system receiving a `baggage` request header SHOULD send it to outgoing requests.
```

Comment:
This seems out of place, it's about retransmitting baggage, not mutating it.

Reply:
This is based on the [respective section for tracestate](https://github.com/w3c/trace-context/blob/master/spec/20-http_request_header_format.md#mutating-the-tracestate-field). If it is considered out-of-place here it may be there as well.

Reply:
I will remove that